tests/cupy_tests/math_tests/test_sumprod.py

```diff
@@ -54,7 +58,7 @@ def test_sum_axis(self, xp, dtype):
     def test_sum_axis_huge(self, xp):
         a = testing.shaped_random((2048, 1, 1024), xp, 'b')
         a = xp.broadcast_to(a, (2048, 1024, 1024))
-        return a.sum(axis=0)
+        return a.sum(axis=2)
```

Comment:
Is this change relevant?

Reply:
Yes, current test has failed.

Reply:
Do you mean it has failed because of memory issue?

Reply:
Issue for this case: https://github.com/cupy/cupy/issues/605

Reply:
OK, so I understand this is not directly related to this PR.
It doesn't have to be split, but I just wanted to confirm so that I can modify the PR title after merge.